### PR TITLE
Bullet component

### PR DESCRIPTION
**Pull request by kim-morgan**
This adds a bullet (at the moment just a small red circle) to the page. When tapped, it moves in the direction of a predefined angle. 

The angle is passed to the Bullet in index.js on line 20. When we get the angle info from the weapon, we can pass it to this.

### CODE REVIEW

File: components/Bullet.js [hunk 1]
```diff
@@ -0,0 +1,55 @@
...
+          uri: "https://pngset.com/images/freetoedit-fireball-gun-fire-effect-flare-light-bonfire-flame-transparent-png-774173.png"
+        }}
+      />
+    </View>
+  )
+}
+
+export default (world, color, angle) => {
+  const initialBullet = Matter.Bodies.circle(300,600,10);
+  const sling = Matter.Constraint.create({
```
Comment (chris-clement) [inline components/Bullet.js:42]: Have a bit of a question on this if you know how it works. What is this sling doing and how does pointA and bodyB  impact the method?

File: entities/index.js [hunk 2]
```diff
@@ -9,6 +16,8 @@ export default restart => {
   world.gravity.y = 0.4;
```
Comment (chris-clement) [inline entities/index.js:16]: One to think about is do we want gravity to begin with? At the moment the bullet starts at the centre of the screen and then drops.

Comment (chris-clement) [reply entities/index.js:16]: Sure thing, just need to figure a way for the bullet to stay in one place until it is fired.

Comment (Hereloss) [reply entities/index.js:16]: The way I'm thinking of it it'll be created upon firing - so shouldn't be an issue

Comment (Riky5) [reply entities/index.js]: it's been commented out for now

File: components/Bullet.js (hunk 1 above)
Comment (Riky5) [reply components/Bullet.js]: removed  it, it was just a previous try to fire the bullet

**Review by chris-clement — APPROVED** [review]

**Pull request merged**